src/routes/DrawerNavigator.tsx

```diff
@@ -1,10 +1,10 @@
-import React, { createContext, useEffect } from "react";
-import { View, Text } from "react-native";
-import { createDrawerNavigator } from "@react-navigation/drawer";
-import CustomDrawer from "./components/CustomDrawer";
-import BeneficiariesNavigator from "./BeneficiariesNavigator";
-import RefencesNavigator from "./ReferencesNavigator";
-import { navigate } from "./NavigationRef";
+import React, { createContext, useEffect, useState } from 'react';
```

Comment:
Please Format the code, we use double quotes, run>
npm run codify 
before you send PR